### PR TITLE
Smoke branch

### PR DESCRIPTION
**Pull request by marco-georgaklis**
# Summary of Changes

Created a new tox environment for executing smoke tests with minimal dependencies

# Related Issues

[1790 (https://github.com/falconry/falcon/issues/1790)]

# Pull Request Checklist

This is just a reminder about the most common mistakes.  Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [ X] Applied changes to both WSGI and ASGI code paths and interfaces (where applicable).
- [X ] Added **tests** for changed code.
- [ X] Prefixed code comments with GitHub nick and an appropriate prefix.
- [ X] Coding style is consistent with the rest of the framework.
- [X ] Updated **documentation** for changed code.
    - [ X] Added docstrings for any new classes, functions, or modules.
    - [ X] Updated docstrings for any modifications to existing code.
    - [ X] Updated both WSGI and ASGI docs (where applicable).
    - [ X] Added references to new classes, functions, or modules to the relevant RST file under `docs/`.
    - [X ] Updated all relevant supporting documentation files under `docs/`.
    - [ X] A copyright notice is included at the top of any new modules (using your own name or the name of your organization).
    - [X ] Changed/added classes/methods/functions have appropriate `versionadded`, `versionchanged`, or `deprecated` [directives](http://www.sphinx-doc.org/en/stable/usage/restructuredtext/directives.html?highlight=versionadded#directive-versionadded).
- [ X] Changes (and possible deprecations) have [towncrier](https://towncrier.readthedocs.io/en/actual-freaking-docs/index.html) news fragments under `docs/_newsfragments/`, with the file name format `{issue_number}.{fragment_type}.rst`. (Run `towncrier --draft` to ensure it renders correctly.)

If you have *any* questions to *any* of the points above, just **submit and ask**! This checklist is here to *help* you, not to deter you from contributing!

*PR template inspired by the attrs project.*

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -20,7 +20,8 @@ envlist = cleanup,
           pytest,
           pytest_sans_msgpack,
           coverage,
-          towncrier
+          towncrier,
+	  minsmoke
```
Comment (CaselIT) [inline tox.ini:24]: missing spaces

**Review by CaselIT — COMMENTED** [review]
we should probably add these to the ci in `.github/workflows`

**Comment by vytas7** [comment]
Hi @marco-georgaklis! Just checking if you're still planning to continue working on this?

**Comment by vytas7** [comment]
Closing this for now, it's been more than one year without any progress.
Let's rebase and reopen if anyone wants to work on this.

**Pull request closed without merging**